### PR TITLE
dsFPD : Enhanced error macro implementation 

### PR DESCRIPTION
**Pull request by Sumip98**
Adding Enhanced error code flag support

### CODE REVIEW

**Review by srinivasgtl — APPROVED** [review]
approving these changes.

**Review by akhilbhas — DISMISSED** [review]
Approved

File: src/test_l1_dsFPD.c [hunk 1]
```diff
@@ -2970,6 +2991,8 @@ int test_l1_dsFPD_register ( void )
 	UT_add_test( pSuite7Segment, "dsGetFPTimeFormat_L1_negative" ,test_l1_dsFPD_negative_dsGetFPTimeFormat );
 	
 
+        extendedEnumsSupported = ut_kvp_getBoolField( ut_kvp_profile_getInstance(), "dsFPD/features/extendedEnumsSupported" );
```
Comment (hari22yuva) [inline src/test_l1_dsFPD.c:2994]: can you point me to the yaml file this

Comment (Sumip98) [reply src/test_l1_dsFPD.c:2994]: Done

File: profiles/sink/Sink_FPD.yaml [hunk 2]
```diff
@@ -1,6 +1,9 @@
 dsFPD:
   Type: sink
   Name: Panel_FPD_1_Indicator
+  features:
+        extendedEnumsSupported: false
```
Comment (hari22yuva) [inline profiles/sink/Sink_FPD.yaml:5]: keep only 2 spaces only

Comment (Sumip98) [reply profiles/sink/Sink_FPD.yaml]: Done

**Review by hari22yuva — APPROVED** [review]
Approved

**Comment by hari22yuva** [comment]
Merged version

**Pull request merged**